<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2074</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto Medium</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="showListButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px; /* Schimbă valoarea pentru mai multă/mică rotunjire */
border: 2px solid #3498db; /* Schimbă culoarea și grosimea conturului */
background-color:#154c79; /* Schimbă culoarea de fundal */
color: white; /* Schimbă culoarea textului */</string>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showListButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px; /* Schimbă valoarea pentru mai multă/mică rotunjire */
border: 2px solid #3498db; /* Schimbă culoarea și grosimea conturului */
background-color:#154c79; /* Schimbă culoarea de fundal */
color: white; /* Schimbă culoarea textului */</string>
    </property>
    <property name="text">
     <string>Customer complains</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showListButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px; /* Schimbă valoarea pentru mai multă/mică rotunjire */
border: 2px solid #3498db; /* Schimbă culoarea și grosimea conturului */
background-color:#154c79; /* Schimbă culoarea de fundal */
color: white; /* Schimbă culoarea textului */</string>
    </property>
    <property name="text">
     <string>Reports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showListButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px; /* Schimbă valoarea pentru mai multă/mică rotunjire */
border: 2px solid #3498db; /* Schimbă culoarea și grosimea conturului */
background-color:#154c79; /* Schimbă culoarea de fundal */
color: white; /* Schimbă culoarea textului */</string>
    </property>
    <property name="text">
     <string>Trip logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px; /* Schimbă valoarea pentru mai multă/mică rotunjire */
border: 2px solid #3498db; /* Schimbă culoarea și grosimea conturului */
background-color:#154c79; /* Schimbă culoarea de fundal */
color: white; /* Schimbă culoarea textului */</string>
    </property>
    <property name="text">
     <string>Manage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showListButton_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px; /* Schimbă valoarea pentru mai multă/mică rotunjire */
border: 2px solid #3498db; /* Schimbă culoarea și grosimea conturului */
background-color:#154c79; /* Schimbă culoarea de fundal */
color: white; /* Schimbă culoarea textului */</string>
    </property>
    <property name="text">
     <string>Fleet</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="removeDriverButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px; /* Schimbă valoarea pentru mai multă/mică rotunjire */
border: 2px solid #3498db; /* Schimbă culoarea și grosimea conturului */
background-color:#154c79; /* Schimbă culoarea de fundal */
color: white; /* Schimbă culoarea textului */</string>
    </property>
    <property name="text">
     <string>Remove driver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addDriverButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px; /* Schimbă valoarea pentru mai multă/mică rotunjire */
border: 2px solid #3498db; /* Schimbă culoarea și grosimea conturului */
background-color:#154c79; /* Schimbă culoarea de fundal */
color: white; /* Schimbă culoarea textului */</string>
    </property>
    <property name="text">
     <string>Add driver</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="driverNameLineEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>1361</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2074</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFleetManager">
    <property name="title">
     <string>FleetEase</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFleetManager"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
